tests/core/math/test_math_funcs.h

```diff
@@ -108,6 +108,23 @@ TEST_CASE_TEMPLATE("[Math] round/floor/ceil", T, float, double) {
 	CHECK(Math::ceil((T)-1.9) == (T)-1.0);
 }
 
+TEST_CASE_TEMPLATE("[Math] integer division round up unsigned", T, uint32_t, uint64_t) {
+	CHECK(Math::division_round_up((T)0, (T)64) == 0);
+	CHECK(Math::division_round_up((T)1, (T)64) == 1);
+	CHECK(Math::division_round_up((T)63, (T)64) == 1);
+	CHECK(Math::division_round_up((T)64, (T)64) == 1);
+	CHECK(Math::division_round_up((T)65, (T)64) == 2);
+	CHECK(Math::division_round_up((T)65, (T)1) == 65);
```

Comment:
I would suggest copy-pasting these same tests also to the next test case template which tests signed integers.

Reply:
Seems like this was done.